ui/src/app/components/runs/run-detail/run-detail.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="content-area">
+  <clr-datagrid (clrDgRefresh)="refresh($event)" [clrDgLoading]="loading">
+    <clr-dg-column>ID</clr-dg-column>
+    <clr-dg-column>Name</clr-dg-column>
+    <clr-dg-column>Type</clr-dg-column>
+    <clr-dg-column>Started</clr-dg-column>
+    <clr-dg-column>Finished</clr-dg-column>
+    <clr-dg-column>Status</clr-dg-column>
+
+    <clr-dg-placeholder>We couldn't find any jobs run!</clr-dg-placeholder>
+
+    <clr-dg-row *ngFor="let jobRun of jobsRun">
+      <clr-dg-cell>{{jobRun.id}}</clr-dg-cell>
+      <clr-dg-cell>{{jobRun.jobName}}</clr-dg-cell>
+      <clr-dg-cell>{{jobRun.jobType.name}}</clr-dg-cell>
+      <clr-dg-cell>{{jobRun.created | date:'medium'}}</clr-dg-cell>
+      <clr-dg-cell>{{jobRun.updated | date:'medium'}}</clr-dg-cell>
+      <clr-dg-cell [ngSwitch]="jobRun.jobStatus.name">
+        <clr-icon *ngSwitchCase="'InQueue'" shape="clock" class="is-solid" style="color: grey"></clr-icon>
```

Comment:
Done

Reply:
I think I would move the the files `dagRunStatuses.constants.ts` and `jobRunStatuses.constants.ts` to `models/enums/dagInstanceStatuses` and `models/enums/jobStatuses` respectively, because these classes are determined by the backend classes `DagInstanceStatuses` and `JobStatuses`. If we change anything in those classes, we will have to change their corresponding frontend classes as well. That's why I think we really need to be strict otherwise we will quickly have inconsistencies between backend and frontend

Reply:
Done